app/src/main/java/ai/elimu/vitabu/ui/storybook/ChapterFragment.java

```diff
@@ -320,6 +320,7 @@ private void playAudioFile(AudioGson audioGson) {
         mediaPlayer = new MediaPlayer();
         try {
             mediaPlayer.setDataSource(audioFile.getPath());
+            mediaPlayer.setPlaybackParams(mediaPlayer.getPlaybackParams().setSpeed(0.5f));
```

Comment:
@gscdev Could you explain why it is necessary to set the speed a second time here, after doing it already with `tts.setSpeechRate(0.5f);` in the `BaseApplication` TTS engine initialization?

Reply:
@nya-elimuai This applies to audios in the word dialog, but maybe it is not necessary because it is only one word. I can remove it.

Reply:
@gscdev Ah, ok, that makes sense. No, I think we should keep the same speed for sounding out words as well, so please keep it as is 👍

However, maybe you can put `0.5f` as a public static final variable somewhere so that we can adjust the number in one location, and have it updated in both places?